react/fields/MonthPicker/MonthPicker.test.js

```diff
@@ -0,0 +1,175 @@
+import chai, { expect } from 'chai';
+import sinon from 'sinon';
+import sinonChai from 'sinon-chai';
+import React from 'react';
+import {
+  createRenderer
+} from 'react-addons-test-utils';
+import { findAllWithClass } from 'react-shallow-testutils';
+import MonthPicker from './MonthPicker';
+
+chai.use(sinonChai);
+
+const renderer = createRenderer();
+
+describe('MonthPicker', () => {
```

Comment:
This test suite doesnt test anything about the value of the monthpicker being set. Any thoughts on getting coverage of this?